### PR TITLE
add terraform RDS DB module

### PR DESCRIPTION
**Pull request by loadtheaccumulator**
Add initial RDS database terraform module for setting up infrastructure

To test...

Defaults are already plugged in so following steps in the [Deploying Infrastructure section](https://github.com/ansible-content-lab/aws_terraform_deployment#deploying-infrastructure) of the [README](https://github.com/ansible-content-lab/aws_terraform_deployment/blob/main/README.md) file works.

### CODE REVIEW

**Comment by jmighion** [comment]
Can you fix the new issues from the linter? `tflint --recursive`

**Review by chrisahl — DISMISSED** [review]
I was able to `terraform apply` fine.  Since you are trying to head out for PTO, I am just going to approve this and later in the new year we can worry about any formatting/naming diffs that we have among files in this repo or diff with azure terraform.

**Comment by loadtheaccumulator** [comment]
> Can you fix the new issues from the linter? `tflint --recursive`

@jmighion  Looking into it.
I also used the Hashicorp plugin for vscode, and checking to see how it differs and if it has a --recursive option to match.

**Comment by loadtheaccumulator** [comment]
Resolved the tflint --recursive and variable = spacing as well.

File: main.tf [hunk 1]
```diff
@@ -10,10 +10,6 @@ locals {
 
 terraform {
   required_providers {
-    aws = {
```
Comment (chrisahl) [inline main.tf:13]: Why did this get removed?

Comment (loadtheaccumulator) [reply main.tf:13]: tflint --recursive threw an item that it wasn't in use by the module.

Comment (loadtheaccumulator) [reply main.tf:13]: the aws work is being done in the vpc and database modules

**Review by chrisahl — APPROVED** [review]

**Pull request merged**